spark-plugins/pom.xml

```diff
@@ -24,7 +24,7 @@
   </parent>
 
   <properties>
```

Comment:
this can actually be removed, it's the same as the parent now that etl-batch and etl-realtime are gone.